ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -276,7 +276,7 @@ func (s *Service) buildUpdateServiceInput(count int64, serviceName, taskDefiniti
 	}
 
 	input := &ecs.UpdateServiceInput{
-		DesiredCount:            aws.Int64(count),
+		DesiredCount:            count,
```

Comment:
The `updateServiceCount` [parameter](https://github.com/aws/amazon-ecs-cli/blob/563e289ac820baf971954d125f186fcdba0593dc/ecs-cli/modules/cli/compose/entity/service/service.go#L634) was changed to accommodate the [updateService](https://github.com/aws/amazon-ecs-cli/blob/563e289ac820baf971954d125f186fcdba0593dc/ecs-cli/modules/cli/compose/entity/service/service.go#L324-L333) use of it. I was trying to avoid any new calls to [describeService](https://github.com/aws/amazon-ecs-cli/blob/563e289ac820baf971954d125f186fcdba0593dc/ecs-cli/modules/cli/compose/entity/service/service.go#L574). Maybe that was a wrong move? Please provide some guidance here as I'm open to suggestions.